PyInstaller/utils/hooks/qt.py

```diff
@@ -662,17 +685,29 @@ def get_qt_network_ssl_binaries(qt_library_info):
     if not ssl_enabled:
         return []
 
+    # Resolve path to the the corresponding python package (actually, its parent directory). Used to preserve directory
+    # structure when DLLs are collected from the python package (e.g., PyPI wheels).
+    package_parent_path = pathlib.Path(qt_library_info.package_location).resolve().parent
```

Comment:
Do we not have to go through the usual drama of case and path separator normalisation and de-duplication? I'm fairly certain that `Path(...).resolve()` only converts relative paths to absolute and flattens out any `./` or `../`s.

Reply:
`Path` itself normalizes the separator, and `Path.resolve` fixes the case for the part of the path that exists on the filesystem:
```
>>> path = pathlib.Path(r'c:\users\rok')
>>> path
WindowsPath('c:/users/rok')
>>> path.resolve()
WindowsPath('C:/Users/Rok')
>>> path = pathlib.Path(r'c:\users\rok2')
>>> path
WindowsPath('c:/users/rok2')
>>> path.resolve()
WindowsPath('C:/Users/rok2')
```